pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -161,6 +162,19 @@ func validateSystemRequirements(mountUtil mount.Interface) (features, error) {
 	return f, nil
 }
 
+// Create a cgroup container manager.
+func createManager(containerName string) *fs.Manager {
+	return &fs.Manager{
+		Cgroups: &configs.Cgroup{
+			Parent: "/",
+			Name:   containerName,
+			Resources: &configs.Resources{
+				AllowAllDevices: true,
+			},
+		},
+	}
+}
+
 // TODO(vmarmol): Add limits to the system containers.
```

Comment:
Later when we handle system containers we would want to replace this manager with my cgroup Manager